src/test/java/org/xstefank/TestUtils.java

```diff
@@ -45,4 +54,17 @@ private static JsonNode createEmptyJsonPayload() {
         return null;
     }
 
-}
\ No newline at end of file
+    public static FormatConfig setUpFormatConfig(SkipPatterns testSkipPatterns) {
+        Format testFormat = new Format();
+        testFormat.setSkipPatterns(testSkipPatterns);
+        FormatConfig testFormatConfig = new FormatConfig();
+        testFormatConfig.setFormat(testFormat);
+        return testFormatConfig;
+    }
+
+    public static void stubMethodWithPowerMock() {
```

Comment:
for a method with a such general name I would expect that it will take an argument stating the name of the stubbed method or that the name would directly incline what is happening inside, when I call this in user code as `TestUtils.stubMethodWithPowerMock()` I cannot possibly know what is happening without checking the content of this method